docs/howto/manage-charms.md

```diff
@@ -11,6 +11,57 @@ Use Charmcraft to quickly initialise your charm project. This generates the
 folder structure, creates placeholder configuration and code files, and
 configures development tooling.
 
+The Charmcraft profiles also all provide the recommended tooling and
+configuration for developing charms. You'll need to install
+[`tox`](https://tox.wiki/en/stable/), and can then run ``tox list`` in the root
+folder to see the available commands.
+
+```{admonition} Best practice
+:class: hint
+
+The quality assurance pipeline of a charm should be automated using a
+continuous integration (CI) system.
+```
+
+```{tip}
+To prepare an environment for running integration tests, such as in CI, use
+the `concierge tool <https://github.com/jnsgruk/concierge>`_ or the
```

Comment:
These links don't seem to render correctly in the preview https://ops--1700.org.readthedocs.build/en/1700/howto/manage-charms.html (ReST vs Markdown issue?)

Reply:
Ah, sorry, I should have previewed the docs here. Yes, the charmcraft docs are ReST and ours are Markdown and I missed converting these. Will do and preview and add a preview link to the description.